src/App.js

```diff
@@ -1,65 +1,198 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
-import Buttons from "./components/Buttons";
-
+import CounterButtons from "./Components/CounterButtons";
+import {useForm} from "react-hook-form";
 
 function App() {
+    const [aardbeien, setAardbeien] = useState(0);
+    const [bananen, setBananen] = useState(0);
+    const [appels, setAppels] = useState(0);
+    const [kiwis, setKiwis] = useState(0);
+
+    function ResetButton() {
+        setAardbeien(0);
+        setBananen(0);
+        setAppels(0);
+        setKiwis(0);
+    }
+
+    const {register, handleSubmit, formState: {errors}} = useForm({
+        mode: "onChange", defaultValues: {
+            "deliveryFrequency" : "every-week",
+            "partOfDay" : "day-time",
+            "firstName" : " ",
+            "lastName" : " ",
+            "age" : "16"
+        }
+    });
 
-    // const plusCounter = () => setCounter(counter + 1);
-    // const minCounter = () => setCounter(counter - 1);
-    // if (counter <= 0) {
-    //     minCounter = () => setCounter(0);
-    // }
+    function onFormSubmit(data) {
+        console.log(data);
+        console.log(`Bestelling: 
+        Aardbeien: ${aardbeien},
+        Bananen: ${bananen},
+        Appels: ${appels},
+        Kiwi's: ${kiwis}`);
+    }
 
     return (
         <>
             <h1>Fruitmand bezorgservice</h1>
+            <section>
+                <CounterButtons
+                    fruitName="🍓 Aardbeien"
+                    counter={aardbeien}
+                    setCounter={setAardbeien}
+                />
+                <CounterButtons
+                    fruitName="🍌 Bananen"
+                    counter={bananen}
+                    setCounter={setBananen}
+                />
+                <CounterButtons
+                    fruitName="🍎 Appels"
+                    counter={appels}
+                    setCounter={setAppels}
+                />
+                <CounterButtons
+                    fruitName="🥝 Kiwi's"
+                    counter={kiwis}
+                    setCounter={setKiwis}
+                />
 
-            <Buttons
-                fruitname="Aardbeien"
-            />
-
-         {/*   <div className="fruit-container">
-                <h2>Aardbeien</h2>
-                <button
-                    type="button"
-                    onClick={() => setCounter(counter - 1)}
-                    disabled={counter === 0}
-                >
-                    -
-                </button>
-                {counter}
                 <button
+                    className="reset-button"
                     type="button"
-                    onClick={() => setCounter(counter + 1)}
+                    onClick={() => ResetButton()}
```

Comment:
Deze kun je iets versimpelen, het is eigenlijk net als rekenen. Als links en rechts van de `=>` lege ronde haakjes staan dan kun je die tegen elkaar wegstrepen 😉
```suggestion
                    onClick={ResetButton}
```